.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,8 +2,6 @@
 
 Add a one line overview of what this PR aims to accomplish.
 
-**Collection**: [Note which collection this PR will affect]
```

Comment:
Let's keep collections please. A one line block determining collections is useful for the regex parser in the changelog builder